src/domain/task-system/services/src/task_executor_impl.rs

```diff
@@ -128,10 +128,21 @@ impl TaskExecutorImpl {
 
     #[tracing::instrument(level = "info", skip_all, fields(task_id = %task.task_id))]
     async fn run_task(&self, task: &Task) -> Result<TaskOutcome, InternalError> {
-        let task_outcome = self
+        // Run task via logical plan
+        let task_run_result = self
             .task_logical_plan_runner
             .run_plan(&task.logical_plan)
-            .await?;
+            .await;
+
+        // Deal with errors: we should not interrupt the main loop if task fails
+        let task_outcome = match task_run_result {
+            Ok(outcome) => outcome,
+            Err(e) => {
+                // No useful task result, but at least the error logged
+                tracing::error!(task=?task, error=?e, "Task run FAILED");
```

Comment:
Let's be consistent in how we log errors at our "entry points":
https://github.com/kamu-data/kamu-cli/blob/8a4b5b49d9997a32f37ccbbe50fd725bd78489fa/src/utils/http-common/src/api_error.rs#L144-L147

Reply:
Corrected in many places, not just this PR